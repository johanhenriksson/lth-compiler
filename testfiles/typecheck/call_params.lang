void callee(int a, int b, int c) { }

int main() {
    callee(1,2,3);
    callee(1,2); // error
    callee(1); // error
    callee(); // error
    main();
}
