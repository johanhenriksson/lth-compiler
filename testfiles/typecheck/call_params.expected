Error at line 5,5: cannot call 'callee': expected 3 arguments, got 2
Error at line 6,5: cannot call 'callee': expected 3 arguments, got 1
Error at line 7,5: cannot call 'callee': expected 3 arguments, got 0
